yarn.lock

```diff
@@ -132,7 +132,7 @@
 
 "@ampproject/remapping@1.0.1":
   version "1.0.1"
-  resolved "https://registry.npmjs.org/@ampproject/remapping/-/remapping-1.0.1.tgz#1398e73e567c2a7992df6554c15bb94a89b68ba2"
+  resolved "https://registry.yarnpkg.com/@ampproject/remapping/-/remapping-1.0.1.tgz#1398e73e567c2a7992df6554c15bb94a89b68ba2"
```

Comment:
This doesn't seem right, can we recreate this yarn.lock file? It should only have the `@types/nodes` changes in it

Reply:
I recreated it, but it still has some updates to `expect-utils` `@type/nodes` `expect` and `jest-matcher-utils`